backend/assessments/2/src/main/java/com/example/springboot/service/OrderService.java

```diff
@@ -0,0 +1,30 @@
+package com.example.springboot.service;
+
+import com.example.springboot.exception.custom.entityNotFoundException;
+import com.example.springboot.model.Orders;
+import com.example.springboot.repository.OrderRepository;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@Service
+public class OrderService {
+    OrderRepository orderRepository;
+
+    @Autowired
+    public OrderService(OrderRepository orderRepository){
+        this.orderRepository = orderRepository;
+    }
+
+    public void placeOrder(Orders order){
+        if(order!=null){
+            orderRepository.save(order);
```

Comment:
Ideally on placing order the products already present in the cart should be marked as placed. The api should not have Order Object passed as request body.